### PR TITLE
Vikings react footer

### PR DESCRIPTION
**Pull request by wanyue0714**
Closes #

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

File: app/javascript/image_sharing/components/Footer.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
 import React, { Component } from 'react';
+import { Row, Col } from 'reactstrap';
+import PropTypes from 'prop-types';
 
 class Footer extends Component {
-  /* Implement your Footer component here */
+  static propTypes = {
```
Comment (shiboying) [inline app/javascript/image_sharing/components/Footer.js]: This is good! And for react, we also want to add associated test, one for `Footer.js` and the other for `App.js`

Comment (wanyue0714) [reply app/javascript/image_sharing/components/Footer.js]: Done with that:
image

Comment (shiboying) [reply app/javascript/image_sharing/components/Footer.js]: Great!

Comment (shiboying) [reply app/javascript/image_sharing/components/Footer.js]: Let's also add a test for App.js

Comment (wanyue0714) [reply app/javascript/image_sharing/components/Footer.js]: Sure! Added

**Pull request merged**